### DISCUSSION TITLE
How to highlight the currently selected node view?

### DISCUSSION THREAD

**Discussion by yellow1912** [ueberdosis/tiptap#1577] [General Questions] [answered]
Hello,

Sorry for asking too many questions. I promise I will write a very detailed show&tell later to help beginners like me to write better custom extensions for tiptap :)

Anyhow, I'm working a custom vuejs node view for my image

![image](https://user-images.githubusercontent.com/374134/125192423-01d7c880-e272-11eb-998c-635bd11ab9d3.png)

As you can see, the image node view is nicely put inside a block to make things easier to view and edit. My question is related to highlighting the currently selected node view. Right now if I click anywhere on the image node view, the cursor is not shown, it's difficult to tell the users that they are editing a certain node view.

For example, below I have the iframe/embed view selected (and no cursor)

![image](https://user-images.githubusercontent.com/374134/125192526-84f91e80-e272-11eb-8705-6d2f611f4d10.png)

Perhaps there is a way to know that certain node view is active/selected so I can add a class or something?

**Comment by yellow1912** [comment, marked as answer]
I found it, selected is the property I look for 

https://www.tiptap.dev/guide/node-views/react/#all-available-props

**Comment by HiChen404** [comment]
But 'selected' is only 'true' when the text is selected, How to get the selected status when the user moves the cursor?

Just like the example below (Lexical.js)：
```ts
  const [isSelected, setSelected, clearSelection] = useLexicalNodeSelection(nodeKey)
```
https://github.com/ueberdosis/tiptap/assets/58675561/e339bc5a-483f-4949-bcd7-0ec90e6be5b4
